libs-scala/resources-grpc/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel_tools:scala.bzl", "da_scala_library", "da_scala_test_suite")
+
+da_scala_library(
+    name = "resources-grpc",
+    srcs = glob(["src/main/scala/**/*.scala"]),
+    tags = ["maven_coordinates=com.daml:resources-grpc:__VERSION__"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//libs-scala/resources",
+        "@maven//:io_grpc_grpc_api",
+        "@maven//:io_netty_netty_common",
+        "@maven//:io_netty_netty_transport",
+    ],
+)
+
+da_scala_test_suite(
+    name = "test",
+    srcs = glob(["src/test/scala/**/*.scala"]),
+    resource_strip_prefix = "libs-scala/resources-grpc/src/test/resources/",
```

Comment:
is this actually required? so far it seemed to work for me to omit this.

Reply:
It works in the sense that it doesn't fail, but if I omit this the `logback-test.xml` file is not picked and I see the full `DEBUG` logging output of Netty.